### DISCUSSION TITLE
I am facing this issue on my mac

### DISCUSSION THREAD

**Discussion by Codebazz** [community/community#34591] [Copilot Conversations] [answered]
Your editor's Copilot extension, version "copilot/1.17.5850", is out of date. Please install a newer version to access this service.

**Comment by davecheney** [comment, marked as answer]
Please update VSCode and then install the latest copilot extension.

**Comment by smit-1999** [comment]
If you are on a Mac and if you face that VS code cant be updated because of read-only drive being active, please move it VS Code to Applications folder and then update it. VSCode, if present in any other folder like Downloads in my case didn't update.
[Reference](https://stackoverflow.com/questions/65422670/why-doesnt-vs-code-auto-update-on-macos). Post the VS code update, you might need to restart your Mac for it to reflect. Reinstall the extension and it should work!
